minidump-stackwalk/README.rst

```diff
@@ -70,5 +70,40 @@ Building with Taskcluster
 The ``build-stackwalker.sh`` script will download a pre-built breakpad
 client binary if possible.
 
-To update this binary, you should use the ``scripts/breakpad-taskcluster.sh``
-script. Follow the directions in the header.
+To update this binary, copy the following task definition into the [TaskCluster
+task creation tool][task-creator]:
+
+```yaml
+created: '2017-04-11T14:17:56.960Z'
+deadline: '2017-04-11T15:17:56.960Z'
+provisionerId: aws-provisioner-v1
+workerType: gecko-misc
```

Comment:
Does this still use the gecko-misc worker?

Reply:
Yup

Reply:
Oh, I see the image now! bd